pages/docs/integrations/grouparoo-onesignal.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: "OneSignal"
+date: "2022-02-18"
+pullQuote: "Learn how to work with the OneSignal Plugin as a Destination in your Grouparoo Reverse ETL pipeline"
+pageNavSelector: "h2"
+---
+
+<PluginDocsLinks plugin="OneSignal" />
+
+## Overview
+
+OneSignal is a customer engagement platform that enables marketers to send push notifications, SMS, and email. Businesses use OneSignal to support business operations with quick and reliable and adaptable communications.
+
+### Destinations
+
+<table className="docTable">
+  <tablebody>
+    <tr>
+      <td>Destination Name</td>
+      <td>Description</td>
+    </tr>
+    <tr>
+      <td>
+        <a href="#onesignal-export-users">
+          <strong>OneSignal Export Users</strong>
+        </a>
+      </td>
+      <td>
+        Enrich OneSignal devices/apps with record and group information as tags.
+      </td>
+    </tr>
+  </tablebody>
+</table>
+
+## Setup
+
+In order to sync data to OneSignal using Grouparoo, you’ll need a OneSignal account. The plan you have through OneSignal may have an effect on the number of contacts you can sync and the amount of tags you can apply.
+
+OneSignal allows you to set up multiple apps on their platform, think of these as projects. To find the App ID and App Key you’ll need for the first steps of configuration, navigate to the specific app where you want to sync data. Choose _Settings_, then _Keys & IDs_ from the top navigation bar. Treat your API key like a password and be sure to keep it private. Anyone with access to this key would be able to send notifications from your OneSignal app.
```

Comment:
> think of these as projects.

What does "projects" mean in this context?

Reply:
That's a good question. I was trying to draw a parallel between Grouparoo's concept of a project and OneSignal's Apps. It actually might not make sense to do that, but it seems like the word App is ambiguous here because some of OneSignal's configurations are "in-app" and some are not. Maybe I could say something more like "think of a OneSignal App as a configuration for sending communications via push notifications, in-app messages, SMS, or email."